src/SetupRemote/SetupRemote.wixproj

```diff
@@ -43,6 +43,10 @@
     <SuppressSpecificWarnings>1076</SuppressSpecificWarnings>
   </PropertyGroup>
   <ItemGroup>
+    <WixExtension Include="WixNetFxExtension">
+      <HintPath>$(WixExtDir)\WixNetFxExtension.dll</HintPath>
```

Comment:
it is needed to get the path to EventLogMessages.dll in .net framework install directory. It is a required field to set while creating event log source.